### PR TITLE
Azure Kinect with OpenCV KinectFusion example

### LINKED ISSUE #357: fastpointcloud example ply missing mesh face triangles (KinFu)
The previous Kinect 2.0 SDK provided us with the ability to export a mesh beside just the point cloud, which seems to be currently missing in the K4A SDK, and as such will make us write more code which the SDK might solve.

If possible please provide the ability/example to generate a vertex colored mesh as well as point cloud.

Or, if this is not an SDK issue, could someone please provide some guidance on what alternative is best used, such as KinFu but its PCL implementation seems to only produce obj/mtl instead of a single ply from RGB images (or is there any other custom solutions for generating a TSDF cloud and a subsequent mesh)?

Thanks in advance, either way, really.

### PR DESCRIPTION
**Pull request by rabbitdaxi**
### Description of the changes:
- An example demonstrate how to use the Azure Kinect SDK with OpenCV and rgbd module from contributor to run KinectFusion

### Before submitting a Pull Request:
- [x] I reviewed [CONTRIBUTING.md](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/blob/develop/CONTRIBUTING.md)
- [x] I [built my changes](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/blob/develop/docs/building.md) locally
- [x] I ran the [unit tests](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/blob/develop/docs/testing.md)
- [x] I ran the [functional tests](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/blob/develop/docs/testing.md) with a device
- [ ] I ran the [performance tests](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/blob/develop/docs/testing.md) with a device

### I tested changes on: 
- [x] Windows
- [] Linux

Since this is newly added example, tested on Windows it is functional (assume all the dependencies are configured correctly)

### CODE REVIEW

File: examples/kinfu/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Azure Kinect - OpenCV KinectFusion Example
+
+## Introduction
+
+The Azure Kinect - OpenCV KinectFusion example shows how to use the Azure Kinect SDK and device with the KinectFusion example from opencv_contrib's rgbd module (https://github.com/opencv/opencv_contrib/tree/master/modules/rgbd). The example demonstrates how to feed calibration and depth images from Azure Kinect APIs to the OpenCV KinectFusion module with undistortion. We render live KinectFusion results in this example as well as generate fused point cloud as a ply file when quitting application.
```
Comment (schultetwin1) [inline examples/kinfu/README.md]: This is a very long paragraph to have on one line. For eadability, can you break this up into multiple lines (of a certain col limit).

Comment (schultetwin1) [inline examples/kinfu/README.md]: Markdown has syntax for putting in links. 
```suggestion
The Azure Kinect - OpenCV KinectFusion example shows how to use the Azure Kinect SDK and device with the KinectFusion example from [opencv_contrib's rgbd module](https://github.com/opencv/opencv_contrib/tree/master/modules/rgbd). The example demonstrates how to feed calibration and depth images from Azure Kinect APIs to the OpenCV KinectFusion module with undistortion. We render live KinectFusion results in this example as well as generate fused point cloud as a ply file when quitting application.
```

**Comment by cdedmonds** [comment]
Broader question. Should this code go here or in the samples repo? @Brent-A

**Comment by rabbitdaxi** [comment]
Sent an offline email, yes, potentially we can move to samples repo if that is a better place to place it

---
In reply to: [497854546](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/pull/389#issuecomment-497854546) [](ancestors = 497854546)

**Comment by rabbitdaxi** [comment]
We will publish this sample code in the Azure-Kinect-Samples github

**Pull request closed without merging**